pkg/cmd/util/docker/docker.go

```diff
@@ -45,3 +45,28 @@ func (h *Helper) GetClientOrExit() (*docker.Client, string) {
 	}
 	return client, addr
 }
+
+// IsAPIVersionCompatible returns true if the current docker API version >= the input
+// version, otherwise it returns false.
+func IsAPIVersionCompatible(dc *docker.Client, version string) bool {
+	env, err := dc.Version()
+	if err != nil {
+		glog.Warningf("Failed to get docker server version: %v", err)
+		return false
+	}
+
+	apiVersion := env.Get("ApiVersion")
+	serverVersion, err := docker.NewAPIVersion(apiVersion)
```

Comment:
Yes: `Unable to parse version "1.8.1.fc21": "fc21" is not an integer`

Upstream fix (internal to kubelet): https://github.com/kubernetes/kubernetes/pull/20827/. We'll probably need to do something similar.